### PR TITLE
Add hello world in pyface.

### PR DESCRIPTION
**Pull request by jonathanrocher**
As a starting point for the pyface introduction and app building, this folder proposes a starting point for the stage 4 of building a minimal pyface app.

### CODE REVIEW

File: stage4.0_first_application/setup.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from os.path import abspath, dirname, isfile, join
+from setuptools import setup, find_packages
+from glob import glob
```
Comment (siddhantwahal) [inline stage4.0_first_application/setup.py:3]: Nit: unused import

File: stage4.0_first_application/pycasa/app/app.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    name = "Pycasa"
+
+    description = "An example Tasks application that explores image files."
+
+    def _task_factories_default(self):
+        return [
+            TaskFactory(
+                id='pycasa.hello_world_task',
+                name="Hellow World Editor",
```
Comment (siddhantwahal) [inline stage4.0_first_application/pycasa/app/app.py:25]:
```suggestion
                name="Hello World Editor",
```

File: stage4.0_first_application/pycasa/ui/tasks/hello_world_pane.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        return "Hello World!"
+
+
+class HelloWorldView(ModelView):
+    model = Instance(HelloWorldModel, ())
+
+    def traits_view(self):
+        return View(
+            Item("model.content")
+        )
```
Comment (siddhantwahal) [inline stage4.0_first_application/pycasa/ui/tasks/hello_world_pane.py:22]: Is the model and model view being redefined intentionally?

**Review by siddhantwahal — APPROVED** [review]
Looks great!

File: stage4.0_first_application/pycasa/ui/tasks/hello_world_pane.py (hunk 3 above)
Comment (jonathanrocher) [reply stage4.0_first_application/pycasa/ui/tasks/hello_world_pane.py]: No, meant to import. Removed.

File: stage4.0_first_application/setup.py (hunk 1 above)
Comment (jonathanrocher) [reply stage4.0_first_application/setup.py]: Fixed. Thanks!

**Comment by jonathanrocher** [comment]
Thank you @siddhantwahal . Sorry I hadn't run flake8 and my editor wasn't configured right. Code is clean and all comments addressed. Merging...

**Pull request merged**